plugins/stack-overflow-backend/src/alpha.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { loggerToWinstonLogger } from '@backstage/backend-common';
+import { readTaskScheduleDefinitionFromConfig } from '@backstage/backend-tasks';
+import {
+  coreServices,
+  createBackendModule,
+} from '@backstage/backend-plugin-api';
+import { searchIndexRegistryExtensionPoint } from '@backstage/plugin-search-backend-node/alpha';
+import { StackOverflowQuestionsCollatorFactory } from './search';
+
+/**
+ * @packageDocumentation
+ * A module for the search backend that exports Stack Overflow modules.
```

Comment:
Not renaming the package for matching the  `search-backend-module-*` pattern, should I create a new package and deprecate this?

Reply:
Yep indeed, this collator belongs in a new `plugins/search-backend-module-stack-overflow-collator` instead, like [the other collators](https://github.com/backstage/backstage/blob/82a480eea68747db2697bd2f0080536a2d8cb995/plugins/search-backend-module-catalog/src/alpha.ts#L63). In doing so, i think you can revert the alpha export changes on this package too right?

Reply:
Done 🙌🏻 : https://github.com/backstage/backstage/pull/20456/commits/3500cdc13200e7c8083725096cfc6222a676bcd9